opm/simulators/wells/MultisegmentWell_impl.hpp

```diff
@@ -1972,4 +1972,41 @@ namespace Opm
     }
 
 
+
+
+    template <typename TypeTag>
+    std::vector<double>
+    MultisegmentWell<TypeTag>::
+    getPrimaryVars() const
+    {
+        const int num_seg = this->numberOfSegments();
+        constexpr int numWellEq = MSWEval::numWellEq;
+        std::vector<double> retval(num_seg * numWellEq);
+        for (int ii = 0; ii < num_seg; ++ii) {
+            const auto& pv = this->primary_variables_.value(ii);
+            std::copy(pv.begin(), pv.end(), retval.begin() + ii * numWellEq);
+        }
+        return retval;
+    }
+
+
+
+
+    template <typename TypeTag>
+    int
+    MultisegmentWell<TypeTag>::
+    setPrimaryVars(std::vector<double>::const_iterator it)
+    {
+        const int num_seg = this->numberOfSegments();
+        constexpr int numWellEq = MSWEval::numWellEq;
```

Comment:
I didn't pay enough attention whilst reviewing this PR.  Our CI system on the other hand states that this object (`numWellEq`) [shadows](https://ci.opm-project.org/job/opm-simulators/1734/gcc/new/) another name.  We should probably rename the object in this local scope to avoid the name clash.